drivers/src/memory_layout.rs

```diff
@@ -28,20 +22,12 @@ pub const DCCM_ORG: u32 = 0x50000000;
 pub const ROM_DATA_ORG: u32 = 0x50000000;
 pub const CFI_STATE_ORG: u32 = 0x500003E4; // size = 6 words
 pub const BOOT_STATUS_ORG: u32 = 0x500003FC;
-pub const MAN1_ORG: u32 = 0x50000400;
-pub const MAN2_ORG: u32 = 0x50001C00;
-pub const FHT_ORG: u32 = 0x50003400;
 pub const LDEVID_TBS_ORG: u32 = 0x50003C00;
 pub const FMCALIAS_TBS_ORG: u32 = 0x50004000;
 pub const RTALIAS_TBS_ORG: u32 = 0x50004400;
```

Comment:
Aren't these addresses part of `PERSISTENT_DATA` as well?

Reply:
They are, but they are also used in the generated linker script here: https://github.com/chipsalliance/caliptra-sw/blob/main/cpu/gen/src/lib.rs#L12-L14.

I've gone ahead and removed them. Let me know if there is any concern, I think it's okay to remove them from the linker script, but I am not sure.

Reply:
lgtm, thanks!